src/main/java/org/folio/circulation/resources/ItemsLostRequiringActualCostResource.java

```diff
@@ -0,0 +1,174 @@
+package org.folio.circulation.resources;
+
+import static org.folio.circulation.domain.ItemStatus.AGED_TO_LOST;
+import static org.folio.circulation.domain.ItemStatus.DECLARED_LOST;
+import static org.folio.circulation.support.AsyncCoordinationUtil.allOf;
+import static org.folio.circulation.support.http.client.CqlQuery.matchAny;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collector;
+
+import org.folio.circulation.domain.Account;
+import org.folio.circulation.domain.FeeFine;
+import org.folio.circulation.domain.ItemLostRequiringActualCostsRepresentation;
+import org.folio.circulation.domain.ItemLostRequiringCostsEntry;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.infrastructure.storage.ServicePointRepository;
+import org.folio.circulation.infrastructure.storage.feesandfines.AccountRepository;
+import org.folio.circulation.infrastructure.storage.feesandfines.FeeFineActionRepository;
+import org.folio.circulation.infrastructure.storage.inventory.ItemRepository;
+import org.folio.circulation.infrastructure.storage.loans.LoanPolicyRepository;
+import org.folio.circulation.infrastructure.storage.loans.LoanRepository;
+import org.folio.circulation.infrastructure.storage.loans.LostItemPolicyRepository;
+import org.folio.circulation.infrastructure.storage.loans.OverdueFinePolicyRepository;
+import org.folio.circulation.infrastructure.storage.users.PatronGroupRepository;
+import org.folio.circulation.infrastructure.storage.users.UserRepository;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.RouteRegistration;
+import org.folio.circulation.support.http.server.JsonHttpResponse;
+import org.folio.circulation.support.http.server.WebContext;
+import org.folio.circulation.support.results.Result;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+public class ItemsLostRequiringActualCostResource extends Resource {
+
+  public final static Collection<String> FIELD_VALUES = List.of(
+    DECLARED_LOST.getValue(),
+    AGED_TO_LOST.getValue());
```

Comment:
This definition is only used in a single method. I think it could it be defined closer to its only usage.